.circleci/config.yml

```diff
@@ -55,6 +55,7 @@ jobs:
       - *restore_cache
       - run: make check-style
       - run: make test
+      - run: make verify-gomod
```

Comment:
Just wondering, why verify after running the tests?

Reply:
Right, this looks odd. It is needed after the tests because we need the modules to be downloaded before they can be verified. "go test" does that. An alternative is to run "go mod download" first to prefill the cache. But that looked like an extra step, and this is simple enough.

Reply:
I see. I guess it wouldn't really be an extra step but the same done explicitly before everything else and not hidden inside the test step. My worry is about the logic of this more than anything. If we verify after `make test` it means waiting for all that process to complete only to discover that it's not valid because some modules mismatched. I'd rather know that from the start, as soon as we downloaded what we need. Does that make sense?

Reply:
That makes sense. Will update :+1: